### PR TITLE
enhance: support db request in Restful api

### PR DESCRIPTION
**Pull request by smellthemoon**
#38077

### CODE REVIEW

**Comment by smellthemoon** [comment]
rerun go-sdk

**Comment by smellthemoon** [comment]
rerun go-sdk

**Comment by smellthemoon** [comment]
rerun go-sdk

File: internal/distributed/proxy/httpserver/handler_v2.go [hunk 1]
```diff
@@ -1468,6 +1475,99 @@ func (h *HandlersV2) createCollection(ctx context.Context, c *gin.Context, anyRe
...
+		return h.proxy.DropDatabase(reqCtx, req.(*milvuspb.DropDatabaseRequest))
+	})
+	if err == nil {
+		HTTPReturn(c, http.StatusOK, wrapperReturnDefault())
+	}
+	return resp, err
+}
+
+// todo: use a more flexible way to handle the number of input parameters of req
+func (h *HandlersV2) listDatabases(ctx context.Context, c *gin.Context, anyReq any, dbName string) (interface{}, error) {
```
Comment (czs007) [inline internal/distributed/proxy/httpserver/handler_v2.go:1514]: unnecessary parameter dbName?

**Pull request merged**